### PR TITLE
maintainers-guide: refresh

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** unknown

## Changes

- [Require remove all co-authors from the result commit message](https://github.com/tldr-pages/tldr/pull/8391#issuecomment-1225909897)
- [Require reviewers wait until all conversations are resolved](https://github.com/tldr-pages/tldr/pull/8392#issuecomment-1225907993)
- Add strict rules when requesting changes is permitted

### CODE REVIEW

**Review by navarroaxel — COMMENTED** [review]
I think this should be placed on https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md. :thinking:

File: contributing-guides/maintainers-guide.md
Comment (navarroaxel) [inline contributing-guides/maintainers-guide.md:92]: All should be removed, 
```suggestion
  Don't forget to clean up the body of the squashed commit message, specially the `Co-authored by:` trailer, giving all the credit to the pull request's author is a good incentive to keep contributing.
  (You can always use the body if you want to include useful information.)
```

File: contributing-guides/maintainers-guide.md
Comment (navarroaxel) [inline contributing-guides/maintainers-guide.md:102]: I like this change.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:91]:
```suggestion
  Don't forget to clean up the body of the squashed commit message, specially the `Co-authored by:` part.
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:92]:
```suggestion
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:98]: I don't see why not, its just more admin work and pointless reviews.

See one of my typo fix commits [here](https://github.com/tldr-pages/tldr/commit/e4719999325f611503c2ad1dc7bea3e8ac25f557)

Does this really need reviewing? IMO, no because no one is going to reject it and no other changes need to be made.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:70]:
```suggestion
  (typos, syntax errors, etc), and the author doesn't respond in a day or two,
```

We should keep this since its not just typos and errors which are trivial, for example, adding a --help command to a page is trivial.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:67]:
```suggestion
  the first maintainer can go ahead and merge it.
```

I don't see the purpose of dismissing, we don't need anyone's opinion (who was requested) before merging, so it just adds more work for the person who merges it and prohibits maintainers, who aren't as active, from reviewing PRs which may have been important however were merged because they had enough reviews.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:59]:
```suggestion
  (2) have all the **review comments addressed**,
```

These two words are almost synonyms, so its just repeating ourselves.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:60]:
```suggestion
  (3) get **approved reviews by at least two maintainers**, (the second maintainer can merge immediately after approving) and
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: contributing-guides/maintainers-guide.md
Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md:70]: All possible trivial changes should be explicitly specified not to confuse maintainers when to use this rule.

File: contributing-guides/maintainers-guide.md
Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md:98]: There is always a human factor. So we can make mistake even in fixing typos. It's sounds funny but I am serious. 😁 If we want allow commiting to the main branch without PRs we have to define what kind of typos we can fix in this way. But maybe there are another solutions to this problem. 🤔 I am going to update this file tomorrow.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:70]: Yes, the word trivial can be interpreted differently of course, however, we could include other things into trivial, one suggestion being grammatical errors

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:91]:
```suggestion
  Don't forget to clean up the body of the squashed commit message, especially the `Co-authored by:` part.
```

File: contributing-guides/maintainers-guide.md
Comment (sbrl) [reply contributing-guides/maintainers-guide.md:67]: Perhaps what is actually mean here is to dismiss *requests for review*?

File: contributing-guides/maintainers-guide.md
Comment (sbrl) [reply contributing-guides/maintainers-guide.md:70]: While being specific is good, but issues holding up a pull request or issue can be many different things that we can't list here exhaustively. For example, not all pull requests are for pages - some PRs are to the client spec, for example, which is often cross-checked by client developers.

File: contributing-guides/maintainers-guide.md
Comment (sbrl) [reply contributing-guides/maintainers-guide.md:98]: I still recommend opening a PR for trivial changes, even if you merge it yourself right after, though I suppose there are exceptions to every rule.

**Comment by EmilySeville7cfg** [comment]
I am going to address all suggestions tomorrow. :)

**Comment by kbdharun** [comment]
@EmilySeville7cfg can we complete this PR soon? 

> I am getting lazy mentioning this PR 😆 to others.

**Comment by EmilySeville7cfg** [comment]
@kbdharun, I hope that we can. :)

File: contributing-guides/maintainers-guide.md
Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md:70]: I've clearly described what changes can be considered as trivial above. ;)

File: contributing-guides/maintainers-guide.md
Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md:98]: Pushes with trivial changes (formal definition is put above) are allowed now.

**Comment by kbdharun** [comment]
@EmilySeville7cfg Can you add the edge cases regarding removing the "Co-authored by" line in PRs. I had made a proposal regarding this in Gitter/Matrix room where I discussed leaving it as it is for PRs with reviews requested and received from Native Speakers (Contributors) and also sometimes we request review from contributors (Like for example we request their review if they created the issue to which a PR is related to) [Something, they make review suggestions without our request, recently we have been seeing it a lot, So I think in that case too we should leave it as it is]. What do you guys think?

**Comment by EmilySeville7cfg** [comment]
It would be great to have a link to this discussion in gitter. :) I can search for it but it's faster just to jump to the first discussion's message.

**Comment by kbdharun** [comment]
> @kbdharun, would be great to have a link to this discussion in gitter. :) I can search for it but it's faster just to jump to the first discussion's message. 

https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$3J-JH39Ft0jNEirQhIGJHZ9-poHnDJzTXBWY8PX2gN4?via=matrix.org&via=gitter.im&via=katyl.info

(I added the perma link for this chat in matrix, I will add the Gitter link soon)

**Comment by waldyrious** [comment]
> @waldyrious What do think regarding this proposal?

If you're talking about your proposal to keep the Co-authored-by lines, I agree with your reasoning, as my thumbs-up in chat reflects :) Not sure if you want me to elaborate on that, but I feel your arguments were complete enough.

If otherwise by proposal you mean this PR as a whole — after taking a cursory look at the diff, I actually would have a few changes to propose to this PR, but honestly I'm going through a particularly busy phase in my work and personal life, so I currently don't have the energy to jump into the detailed discussion that has already taken place above. That said, I do agree with the overall direction of making things more explicit, so I'll defer to the others' judgment on this, and maybe later I will revisit this document and propose changes.

If there's any particular point of contention you'd like my opinion on, though, do let me know.

**Comment by EmilySeville7cfg** [comment]
@CleanMachine1, please re-review changes.

File: contributing-guides/maintainers-guide.md
Comment (kbdharun) [inline contributing-guides/maintainers-guide.md:91]:
```suggestion
  (You can always use the body if you want to include useful information.)
  
 Note: For some PRs with reviews and proofreading from contributors or native speakers outside of the tldr -pages organization the `Co-authored by` part must be included in the merge commit message.
```

Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md:91]: Do u want to tell that for external contributors such `Co-authored` lines should be left in other words?

**Comment by EmilySeville7cfg** [comment]
According to our [maintainers guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/maintainers-guide.md#ii-handling-prs) @CleanMachine1 review should be dismissed and:

> If a PR fails to get a review from a second maintainer after a few days, the first maintainer should ping others for review. If it still lingers around for over a week without a second maintainer’s approval, the first maintainer can go ahead and merge it.

File: contributing-guides/maintainers-guide.md
Comment (kbdharun) [reply contributing-guides/maintainers-guide.md:91]: > Do u want to tell that for external contributors such `Co-authored` lines should be left in other words?

Yes, Not for us `Collaborators/members` but for other contributors.

File: contributing-guides/maintainers-guide.md
Comment (kbdharun) [inline contributing-guides/maintainers-guide.md:93]:
```suggestion
 Note: For some PRs with reviews and proofreading from contributors or native speakers outside of the tldr-pages organization the `Co-authored by` part must be included in the merge commit message.  
```
Adding 2 spaces to make [`Line 94`](https://github.com/tldr-pages/tldr/blob/fee7444ea1349418f91b9fe7cb388b63418fa60c/contributing-guides/maintainers-guide.md) go to next line

Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md]: done ;)

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:63]:
```suggestion
  These rules mean that trivial PRs can be merged before the default 24 hours.
```

Currently not grammatically correct.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:61]:
```suggestion
  (4) have been open for at least **12 hours** unless the changes are trivial ( E.G. contains only typo fixes or syntax error fixes)
```

There is just no need to so specific, I think it can be left the maintainers digression.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:68]:
```suggestion
  the first maintainer can go ahead and merge it.
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:69]:
```suggestion
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [reply contributing-guides/maintainers-guide.md:68]: I don't see the point in removing the requested users, they will still have the notification they were pinged (to my knowledge) and its just extra admin work for the person who merges.

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:77]: Not currently happy with this wording, maybe someone else can help fix it because it isn't very clear IMO

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:82]:
```suggestion
  It doesn't mean that maintainers reviewing PRs from new contributors should allow violation to the 
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:84]:
```suggestion
  with more detail.
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:87]:
```suggestion
  Grammatical errors and typos also can be a reason to request changes. No other reasons are valid to do so.
```

File: contributing-guides/maintainers-guide.md
Comment (CleanMachine1) [inline contributing-guides/maintainers-guide.md:96]:
```suggestion
  Note: For all PRs from external contributors (outside of the tldr-pages organization), the `Co-authored by` part must be included in the merge commit message.  
```

File: contributing-guides/maintainers-guide.md
Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md:77]: Can you tell me what exactly u don't like in the current wording?

File: contributing-guides/maintainers-guide.md
Comment (EmilySeville7cfg) [reply contributing-guides/maintainers-guide.md]: I've written that it's up to maintainers to decide what is trivial or not. It means that everything depends on reviewers who review some PR.

File: contributing-guides/maintainers-guide.md
Comment (kbdharun) [reply contributing-guides/maintainers-guide.md:77]: I think the current wording is fine. Not sure what to change in it.

**Pull request closed without merging**
